packages/services/rate-limit/src/index.ts

```diff
@@ -24,67 +23,28 @@ async function main() {
     });
   }
 
-  const server = await createServer({
-    name: 'rate-limit',
-    tracing: false,
-    log: {
-      level: env.log.level,
-    },
-  });
-
-  try {
-    const ctx = createRateLimiter({
-      logger: server.log,
-      rateLimitConfig: {
-        interval: env.limitCacheUpdateIntervalMs,
-      },
-      rateEstimator: env.hiveServices.usageEstimator,
-      emails: env.hiveServices.emails ?? undefined,
-      storage: {
-        connectionString: createConnectionString(env.postgres),
-      },
-    });
+  const logger = createLogger() as FastifyLoggerInstance;
```

Comment:
If you are dropping the `fastify` dependency, I think you should also get rid of using the `FastifyXYZ` types?

Reply:
yes, the only concern would be to ensure that all services have been migrated to `whatwg` because I think there maybe is 1 or 2 that are still using fastify which would cause a type error when we change the arg type of `logger` for `registerShutdown` in `services/service-common`

Reply:
I see. I'm also fine with grouping all these PRs into one if they are all similar and does the same thing - and then we can just drop `fastify` types in the same PR?

Reply:
I was planning to open a single PR with everything but @n1ru4l advised to open a PR per service.

<img width="369" alt="image" src="https://user-images.githubusercontent.com/72823042/209670694-18c90558-056e-416d-93fb-cfb5eeb52164.png">